### ISSUE TITLE
Time zone incorrect in charts

### ISSUE THREAD

**Issue by SlimeDog** [plan-player-analytics/Plan#2780]
### Describe the issue

The timezone shown in performance reports is UTC, despite the configuration specification
```
Formatting:
    Dates:
        TimeZone: 'server'
```
Current server time is
```
$ date
Fri Dec  9 18:44:59 PST 2022
```
Screen shot displays (righthand) dates on 2022-12-10 02:00

![Screen Shot 2022-12-09 at 6 45 12 PM](https://user-images.githubusercontent.com/17748923/206825026-64222041-fc82-4165-8148-9f136042614b.png)

### Plugin versions

```
>version
[18:45:52] [Server thread/INFO]: This server is running CraftBukkit version 3610-Spigot-6198b5a-19df23a (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT)
>version Plan
[18:45:30] [Server thread/INFO]: Plan version 5.5 build 2121
[18:45:30] [Server thread/INFO]: Author: AuroraLS3
```

**Comment by AuroraLS3** [comment]
Ahh I'll need to check if I remembered to put in the timezone offset in the React parts - thanks for the report!

**Comment by SlimeDog** [comment]
Thanks. It looks like the `Formatting.Dates.` values aren't being used, either.

**Comment by SlimeDog** [comment]
Done: https://github.com/plan-player-analytics/Plan/issues/2781

**Comment by AuroraLS3** [comment]
Build where this is fixed will appear here in ~15 minutes https://github.com/plan-player-analytics/Plan/actions/runs/3663368129

**Comment by SlimeDog** [comment]
Confirmed fixed in CI build 5.5-b2131. Thanks.
